coda/templates/mdstore/bag_info.html

```diff
@@ -1,92 +1,92 @@
+                    <i class="icon-globe"></i> <a href="http://texashistory.unt.edu/{{ v }}">{{ v }}</a>
+                {% elif k == 'Payload-Oxum' %}
+                    <i class="icon-hdd"></i> {{ payload_oxum_size }} bytes, <i class="icon-file-alt"></i> {{ payload_oxum_file_count }} files
+                {% elif k == 'Bagging-Date' %}
+                    <i class="icon-calendar"></i> {{ bag_date|timesince }} ago
+                {% elif k == 'Organization-Address' %}
+                    <i class="icon-building"></i> {{ v }}
+                {% elif k == 'Contact-Phone' %}
+                    <i class="icon-phone"></i> {{ v }}
+                {% elif k == 'BagIt-Version' %}
+                    <i class="icon-info-sign"></i> {{ v }}
+                {% elif k == 'Bag-Size' %}
+                    <i class="icon-hdd"></i> {{ v }}
+                {% elif k == 'Contact-Name' %}
+                    <i class="icon-user"></i> {{ v }}
+                {% elif k == 'Internal-Sender-Identifier' %}
+                    <i class="icon-tag"></i> {{ v }}
+                {% elif k == 'Source-Organization' %}
+                    <i class="icon-home"></i> {{ v }}
+                {% else %}
+                    <i class="icon-leaf"></i> {{ v }}
+                {% endif %}
+            </td>
+        </tr>
+    {% endfor %}
+</table>
+<!-- EVENTS HEADER -->
+{% if linked_events|length > 0 %}
+
+<h3><i class="icon-link icon-large"></i>There are <a href="http://{{ request.META.HTTP_HOST }}/event/search/?linked_object_id={{ bag }}">{{ total_events }} premis events</a> associated with {{ bag }}:</h3>
+
+
+<table class="table table-striped">
+  <thead>
+    <tr>
+        <th>Event ID</th>
+        <th>Event Date</th>
+        <th>Event Status</th>
+        <th>Linked Object(s)</th>
+        <th>Classified Type</th>
+	</tr>
+  </thead>
+  <tbody>
+
+{% for event in linked_events %}
+
+    <tr>
+
+        <td><i class="icon-tag"></i><a href="http://{{ request.META.HTTP_HOST }}/event/{{ event.identifier }}"> {{ event.identifier }}</a></td>
+
+
+        <td>{{ event.date }}</td>
+
+
+        <td>{% if event.outcome|slice:"53:" == 'success' %} <a href="http://{{ request.META.HTTP_HOST }}/event/search/?outcome={{ event.outcome|urlencode }}"><span class="label label-success">Success</span></a>{% else %} <a href="http://{{ request.META.HTTP_HOST }}/event/search/?outcome={{ event.outcome|urlencode }}"><span class="label label-important">Failure</span></a>{% endif %}
+        </td>
+
+        <td><i class="icon-link"></i> {{ event.linked_objects }}</td>
+
+        <td><i class="icon-asterisk"></i> <a href="http://{{ request.META.HTTP_HOST }}/event/search/?type={{ event.event_type|urlencode }}"> {{ event.event_type }}</a></td>
+
+    </tr>
+
+
+{% endfor %}
+	</tbody>
+</table>
+
+
+{% else %}
+    <h3><i class="icon-remove icon-large"></i> There are no <em>premis events</em> associated with {{ bag }}.</h3>
+{% endif %}
+{% endblock %}
```

Comment:
Do you know why this file shows so many changes even though it looks like there is only one line changed? Is it endline chars being switched or something?

Reply:
I made the autocrlf setting in my IDE before pushing this change. Not sure why it's showing so many changes. I'll look into it.